src/module/actor/base.ts

```diff
@@ -543,6 +543,7 @@ class ActorPF2e<TParent extends TokenDocumentPF2e | null = TokenDocumentPF2e | n
 
     protected override _initialize(): void {
         this.constructed ??= false;
+        this._itemTypes = null;
```

Comment:
I don't know if handling it here is safe. _itemTypes is a cache, so it needs to be killed every data prep, not just the first.

Reply:
`_initialize` is what calls `prepareData`

Reply:
Seems to indeed be the case after testing, but that does mean the comment is possibly wrong?

> Perform one-time initialization tasks which only occur when the Document is first constructed.

Reply:
Since the only thing that gave me pause is foundry docs being a liar liar pants on fire I can push button